contracts/README.md

```diff
@@ -2,57 +2,54 @@
+TokenSpoke contracts allow for scaling token amounts, which should be used when the spoke asset has a higher or lower denomination than the hub asset, such as allowing for a ERC20 hub asset with a denomination of 6 to be used as the native EVM asset on a spoke chain (with a denomination of 18).
 
-All messages sent by `TeleporterTokenDestination` instances are sent to the specified token source contract, whether they are to redeem the collateral on the home or chain or route the tokens to another destination chain. Routing tokens from one destination chain to another is referred to as a "multi-hop", where the tokens are first sent back to their token source contract to update its accounting, and then automatically routed on to their intended destination.
+### `ERC20TokenSpoke`
+A concrete implementation of `TokenSpoke`, `IERC20TokenBridge`, and `IERC20` that handles the minting and burning of an ERC20 spoke chain asset. Note that the `ERC20TokenSpoke` contract is an ERC20 implementation itself, which is why it takes the `tokenName`, `tokenSymbol`, and `tokenDecimals` in its constructor. All of the ERC20 interface implementations are inherited from the standard OpenZeppelin ERC20 implementation, and can be overriden in other implementations if desired.
 
-`TeleporterTokenDestination` contracts allow for scaling token amounts, if the representative "wrapped" token is not a 1-to-1 equivalent of the backing asset. This token scaling can be used when the destination chain asset has a higher or lower denomination than the source asset, such as allowing for a ERC20 source asset with a denomination of 6 to be used as the native EVM asset on a destination chain (with a denomination of 18).
+### `NativeTokenSpoke`
+A concrete implementation of `TokenSpoke`, `INativeTokenBridge`, and `IWrappedNativeToken` that handles the minting and burning of the native EVM asset on the spoke chain using the native minter precompile. Deployments of this contract must be given the permission to mint native coins in the chain's configuration. Note that the `NativeTokenSpoke` is also an implementation of `IWrappedNativeToken` itself, which is why the `nativeAssetSymbol` must be provided in its constructor. By default, `NativeTokenSpoke` instances have a denomination of 18, which is the denomination of the native asset of EVM chains.
```

Comment:
By default meaning the default implementation. Anyone can supply their own implementation of the spoke contract and change the 18.

Reply:
if someone wanted to supply their own implementation, they'd want to inherit ours, ideally, right? to get burned tx fee reporting, native asset supply handling, send-and-call handling, etc, i'd think. if they just copy the code, sure, change the 18, but if they want to inherit our code then there's no easy way to change the 18, since it's hardcoded in the `TokenSpoke()` constructor call. (right? or am i missing something?)

also, this statement in the doc is about the `NativeTokenSpoke` implementation, not some other implementation. if you copied the code and changed the 18, that wouldn't be a `NativeTokenSpoke` implementation anymore. Unless I'm missing something (maybe some diamond pattern type of inheritance of the `NativeTokenSpoke` code but overriding the underlying base class? i just made that up 🫣 ) there's no way to get anything besides 18 in an instance of the `NativeTokenSpoke` contract.

Reply:
Removed the "by default" 👍